solr/CHANGES.txt

```diff
@@ -256,9 +256,8 @@ when told to. The admin UI now tells it to. (Nazerke Seidan, David Smiley)
 * SOLR-15209: The old LegacyAssignStrategy has been refactored into the SimplePlacementPlugin. This is still the default
   placement policy for Solr. (Houston Putman, Ilan Ginzburg)
 
-* SOLR-15257: Add DocSetQuery in lieu of DocSet.getTopFilter. This issue led to removing
-  Filter.java and eliminating all dependencies on its main method getDocIdSet. With Filter deleted ,
-  and replaced with Lucene's Query.java, the changes might spark some performance optimizations (Collins Abanda, Mike Drob, Tim Potter)
+* SOLR-12336: Remove Filter and SolrFilter, replaced by DocSetQuery which is a Query
```

Comment:
Instead, I'd prefer two JIRAs to reference in the CHANGES.txt.  One is DocSet.getTopFilter being replaced by DocSet.makeQuery.  This is one discrete change, even though you're choosing to do so in one commit/PR including other stuff.  The other is that Filter no longer has a need to exist due to multiple JIRA issues, implemented by a number of issues.  Perhaps reference nobody in particular there, or reference all of us at your discretion.

Reply:
Ok, made the edit referencing two separate JIRAs: SOLR-15257 and SOLR-12336 👍

Reply:
I don't think the summary of SOLR-12336 (removal of Filter) shouldn't reference DocSetQuery; that had its own issue (SOLR-15257).  Also, the cast of characters, if you look at the JIRAs referenced from this umbrella issue, included Michael Gibney (SOLR-14185).  I suppose in this one you might mention SolrConstantScoreQuery's removal too.